config/fxdata/creature.cfg

```diff
@@ -83,6 +83,7 @@ Graphics = ATTACK
 RangeMax = 284
 Properties = DESTRUCTIVE MELEE_ATTACK
 Function = creature_fire_shot SHOT_SWING_SWORD 0
+InstantFPCast = 0
```

Comment:
Could you please rename this to FPInstantCast and move it right below FPResetTime? For all instances please.